bin/update_hosts.sh

```diff
@@ -3,28 +3,29 @@ set -e
 
 : "${HOSTS_FILE:=/etc/hosts}"
 : "${COREFILE:=services/coredns/Corefile}"
-: "${MORPH_CHAIN_CONFIG:=services/morph_chain/protocol.privnet.yml}"
+: "${NEOFS_CHAIN_CONFIG:=services/ir/cfg/config.yml}"
 temp_file=$(mktemp)
 
 # Get default hosts
 make hosts > "$temp_file"
 
-# Get the morph-chain IP address from the $COREFILE file
-morph_chain_ip=$(grep "morph-chain.neofs.devenv" "$temp_file" | awk '{print $1}')
+# Get the NeoFS chain IP address from the $COREFILE file
+neofs_chain_ip=$(grep "ir01.neofs.devenv" "$temp_file" | awk '{print $1}')
 
 # Replace the IP address in the $COREFILE file
-sed -i -E "s/(nns[[:space:]]*http\:\/\/)([0-9]{1,3}[.]){3}[0-9]{1,3}(:30333)/\1$morph_chain_ip\3/" "$COREFILE"
+sed -i -E "s/(nns[[:space:]]*http\:\/\/)([0-9]{1,3}[.]){3}[0-9]{1,3}(:30333)/\1$neofs_chain_ip\3/" "$COREFILE"
 
 # Get the line numbers of "Addresses:"
-addresses_lines=$(grep -n "Addresses:" "$MORPH_CHAIN_CONFIG" | cut -d ':' -f 1)
+# FIXME: grep by 'listen:' is unstable, jump to exact YAML fields
```

Comment:
Likely can be fixed with some `jq` incantation.

Reply:
do you mind fixing this separately? the approach was the same before

Reply:
> Likely can be fixed with some `jq` incantation.

The jq is sometimes not installed on the custom runners.

Reply:
> do you mind fixing this separately

Issue?

> The jq is sometimes not installed

We already use it in scripts, so I doubt dev-env would work without it.

Reply:
@roman-khimov #302 (and #303)